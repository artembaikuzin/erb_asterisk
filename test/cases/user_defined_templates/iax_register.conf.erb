<%= render 'iax_reg', login: '123', host: 'localhost', password: 'qwerasdf' %>
