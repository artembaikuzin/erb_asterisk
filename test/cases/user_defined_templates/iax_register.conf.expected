register => 123:qwerasdf@localhost

